### ISSUE TITLE
Gearswap: request to ignore outgoing text that is blocked upstream

### ISSUE THREAD

**Issue by bamboooya** [Windower/Lua#2218]
Gearswap is interfering with an upstream addon I'm working on.  I have Gearswap configured to run downstream from my addon.  My addon sometimes modifies outgoing text and sets `blocked`  to true only for the modifications to be negated by gearswap afterwards.  Would updating Gearswap to preserve outgoing text when it is blocked upstream interfere with Gearswap behavior in any way?

Specifically, I would like to change https://github.com/Windower/Lua/blob/e2a0ae23a0c002dae59cd036eb33f3edfcb3068b/addons/GearSwap/triggers.lua#L45 to:
```
if gearswap_disabled or blocked then return modified end
```

Thank you for this awesome addon and please let me know if it is ok to fork and create a PR for this?

**Comment by RubenatorX** [comment]
Due to limitations in the way that events are handled in windower4, this is not possible to do.
The order in which addons individually handle events is undefined, and so there's no way to reliably force one addon to process an event before another. So one cannot reliably force one addon to be processed upstream or downstream of another addon.
There are plans to fix this problem for future versions of windower, but in the meantime, I don't think this change could be accepted.

**Comment by bamboooya** [comment]
Thank you for the prompt follow up on this.  I guess I made a possibly incorrect assumption on the event handling order of addons based on what I observed during testing.  From what I've seen, the event handling order is tied to the order that the addons are loaded (i.e., when loading my addon before Gearswap, my addon's event seemed to be triggered first).

I have a hope of eventually making my addon available to others, hence reaching out to see if this minor change could negatively impact Gearswap functionality.  Currently my addon is very beta level in maturity and this is one of several limitations I have run into.  The other more significant limitation is a lack of fast/optimized window size, position, focus, etc. manipulation.  If it's not possible to accept this change, we can close this issue.  Thanks again.
